contracts/bonding/BondingManager.sol

```diff
@@ -498,41 +529,133 @@ contract BondingManager is IBondingManager, Controllable {
         return transcoderPools.transcoderStake(_transcoder);
     }
 
+    function transcoderRates(address _transcoder) public constant returns (uint256, uint256, uint256, uint256, uint256, uint256) {
```

Comment:
Let's add a comment for documentation generation purposes about what people can expect the 6 return values to be.